protocol/protocol.tex

```diff
@@ -10868,9 +10871,13 @@
   \item $\ExtractPbot\big(P \typecolon \GroupP\big) = \ExtractP(P)$.
 \end{formulae}
 
-\vspace{-3ex}
-\nnote{$\ExtractP$ returns the type $\GroupPx$ which is precise for its range, unlike $\ExtractJ$
-which returns a bit sequence.}
+\vspace{-2ex}
+\begin{pnotes}
+  \item There is no solution to $y^2 = 0^3 + 5$ in $\GF{\ParamP{q}}$, and so $\ExtractP(P)$
+        can only be $0$ when $P = \ZeroP$.
```

Comment:
We need to say this in order to argue that MerkleCRH<sup>Orchard</sup> only returns 0 for the exceptional case that we proved would yield a discrete log.